src/views/v2/month/calendar-body/day/multiday-events/multiday-event/bar.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * View: Month View - Multiday Event Bar
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/month/calendar-body/day/multiday-events/multiday-event/bar.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ * @var WP_Post $event An event post object with event-specific properties added from the the `tribe_get_event`
+ *
+ * @see tribe_get_event() For the format of the event object and its properties.
+ *
+ */
+
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 2731d01f1cdfcb9634444692f11416d5</sub>